docs/testing/releases/490.md

```diff
@@ -0,0 +1,109 @@
+- Setup the extension with a G-X type ID from your GA account 
+- Install [Google Analytics Debugger](https://chrome.google.com/webstore/detail/google-analytics-debugger/jnkmfdileelhofjcijamephohjechhna?hl=en) extension and enable it
+- Go to the store and trigger some events using the All Products Block and Cart block (feature plugin only). See list of events in https://github.com/woocommerce/woocommerce-gutenberg-products-block/pull/3953.
+- The console will show when events are fired, and the Google Analytics Debugger will also show logs.
+
+To test SSR blocks:
+
+1. Insert product grid block into a page
+2. View the page. Check a view_item_list event fired in GA.
+
+To test the checkout functionality (feature plugin only):
+
+1. Visit checkout with items in your cart. You will see a begin_checkout event
+2. Start editing fields. Depending on the field you edit (which step) you will see a checkout_progress event. e.g. adding billing information will add checkout_progress with a number indicating the step you are on.
+3. Select a shipping rate. See that a set_checkout_option event fired.
+4. Select a payment method. See that a set_checkout_option fired.
+5. Leave a required field blank and place order. See _exception_ event.
+6. Place order. See add_payment_info event.
+
+## Feature plugin only
+
+### General smoke testing
+
+The following are related to various changes impacting some existing flows so just need validated there's nothing broken:
+
+-   Cart and checkout coupons interaction works as expected.
+-   Cart and checkout changing quantities works as expected.
+-   Cart and checkout shipping interaction works as expected.
+-   Place orders via the Checkout block starting from the Cart Block & shortcode.
+-   Test payment methods work as expected.
+-   Test express payment methods work as expected.
+
+:*Testing steps of these PRs are included in this section: Fix Circular Dependencies During Builds (#4025), Remove useCheckoutRedirectUrl() (#4032) and  Update @automattic/data-stores and remove mapped-types.ts (#4024)*
+
+### Load translation file for JS files that has translatable strings (#4050))
+
+- In WooCommerce > Settings check the option to Enable tax rates and calculations.
+- In the Tax tab, set `Display prices during cart and checkout` to `Excluding tax` and `Display tax totals` to `As a single total`.
+- In the Standard rates tab, make sure you have at least one tax rate created. If not, create it.
+- In Settings > General, change site language to something other than English; Spanish is a good example of a 100% translated language.
+- Go to Dashboard > Updates and scroll down to download the new translations immediately.
+- Visit Cart or Checkout block.
+- Strings like `Taxes` should be translated (in Spanish, it should be `Impuestos`).
+
+### Fix TwentyX styling issues for cart/checkout form fields (#4046)
```

Comment:
```suggestion
### Fix Twenty(X) styling issues for cart/checkout form fields (#4046)
```

I think adding brackets makes it a bit more clear the "X" is just a placeholder.

Reply:
Done in 0cc8945631a1954b048caeaa12a8c1c730bd6951. :slightly_smiling_face: